index.html

```diff
@@ -310,16 +307,13 @@
 When <code>cancelIdleCallback(handle)</code> is called, the user agent MUST run the following steps:
 </p>
 <ol>
-<li>
-Let <var>document</var> be the <code>Window</code> object's
-<a href="http://www.w3.org/TR/html5/browsers.html#active-document">active document</a> object. [[!HTML5]]
-</li>
-<li>Find the entry in either the <var>document</var>'s <a>list of idle request callbacks</a> or the <var>document</var>'s <a>list of runnable idle callbacks</a> that is associated with the value <var>handle</var>.</li>
-<li>If there is such an entry, remove it from both the <var>document</var>'s <a>list of idle request callbacks</a> and the <var>document</var>'s <a>list of runnable idle callbacks</a>.
+<li>Let <var>window</var> be this <code>Window</code> object.</li>
+<li>Find the entry in either the <code>window</code>'s <a>list of idle request callbacks</a> or <a>list of runnable idle callbacks</a> that is associated with the value <var>handle</var>.</li>
```

Comment:
Oh, there's also a couple `the <var>window</var>`, which should be `<var>window</var>` or `this <code>Window</code> object`